client/web/src/insights/src/core/analytics.ts

```diff
@@ -1,6 +1,6 @@
 import { isEqual } from 'lodash'
-import { isSettingsValid, Settings, SettingsCascadeOrError } from '../../../shared/src/settings/settings'
-import { TelemetryService } from '../../../shared/src/telemetry/telemetryService'
+import { isSettingsValid, Settings, SettingsCascadeOrError } from '@sourcegraph/shared/out/src/settings/settings'
+import { TelemetryService } from '@sourcegraph/shared/out/src/telemetry/telemetryService'
```

Comment:
```suggestion
import { isSettingsValid, Settings, SettingsCascadeOrError } from '@sourcegraph/shared/src/settings/settings'
import { TelemetryService } from '@sourcegraph/shared/src/telemetry/telemetryService'
```

Oh no, this happened again. I start wondering if this started when we switched to having the symlink to these packages 😞

Reply:
Yeah, I started this PR even before PR where we've seen this problem as well, So since that I marked out folder as none import directory in my Webstorm IDE. 

> switched to having the symlink to these package

Agree, probably that might be the reason. But I still think this is just my IDE settings.